rust/azure_iot_operations_services/src/schema_registry.rs

```diff
@@ -91,10 +91,10 @@ pub struct ServiceError {
 #[builder(setter(into), build_fn(validate = "Self::validate"))]
 pub struct GetRequest {
     /// The unique identifier of the schema to retrieve. Required to locate the schema in the registry.
-    id: String,
+    pub id: String,
```

Comment:
since this one has validate, do we need to add this validation later on as well in case it gets changed?

Reply:
Oh, I'll take this out then.

Reply:
If we think that this is going to be something that scales out in the future, maybe getters would be preferable.

Reply:
Advantage to validating before the service is avoiding the network call. But whether we validate in build vs get() seems fairly similar to me if we need the mutability. Agree that getters/setters would also solve the same issue since set could include the validation